### PR TITLE
Update Dockerfile to use TF 2.4

### PR DESCRIPTION
**Pull request by amygdala**
retested a fresh image build again with the mnist notebook; all looks good.

### CODE REVIEW

File: docker/terra_image/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ USER root
 ENV PIP_USER=false
 
 RUN pip3 install -U git+https://github.com/broadinstitute/ml4h.git
+RUN pip3 install -U tensorflow==2.4.3
```
Comment (deflaux) [inline docker/terra_image/Dockerfile:8]: Can you please add a comment about when these overrides should be removed?

e.g., remove them after https://github.com/DataBiosphere/terra-docker/issues/244 ?

Thanks!

**Review by deflaux — APPROVED** [review]
Looks good! Please just add one more comment.

File: docker/terra_image/Dockerfile (hunk 1 above)
Comment (amygdala) [reply docker/terra_image/Dockerfile:10]: oops, had meant to do that.  Done.

Comment (deflaux) [reply docker/terra_image/Dockerfile:10]: thx!

**Pull request merged**